Assignments/Program.cs

```diff
@@ -12,35 +13,34 @@ public class T15 {
    public static void Main () {
       Console.WriteLine ("\x1B[4m" + "Check for Isogram Word:-" + "\x1B[0m");
       Console.WriteLine ("An isogram is a word that has no duplicate letters.");
-      string output;
-      List<string> testCases = new () { "apple", "banana", "cat", "doctrine", "elephant", "fruits", "pebble", "rubiks", "yellow", "zebra" };
+      List<string> testCases = new () { "", "apple", "banana", "cat", "doctrine", "elephant", "fruits", "pebble", "rubiks", "yellow", "zebra", "12345", "123345", "@#$4%! -)", "*765^%$*" };
       Console.WriteLine ();
       Console.WriteLine ("\x1B[4m" + "Test Cases:" + "\x1B[0m");
       for (int i = 0; i < testCases.Count; i++) {
-         Console.Write ($"{i + 1}. {testCases[i]} - ");
-         (output, Console.ForegroundColor) = IsIsogram (testCases[i]) ? ("Isogram", Green) : ("Not an Isogram", Red);
-         Console.WriteLine ($"{output}");
-         Console.ResetColor ();
+         var t = testCases[i];
+         Console.Write ($"{i + 1, 2}. {t, 11} - ");
+         perform (t);
       }
       Console.Write ("\nNow, let's enter a word: ");
-      string input = Console.ReadLine ().ToUpper ();
-      (output, Console.ForegroundColor) = IsIsogram (input) ? ("Isogram", Green) : ("Not an Isogram", Red);
+      perform (Console.ReadLine ());
+   }
+
+   /// <summary>Prints the result whether the input is isogram or not</summary>

```

Comment:
![image](https://github.com/vikrampg1710-tmm/Training/assets/75829713/f0c03be0-5a2a-4705-8724-df23b125323f)
You haven't turned on the option for Documentation file. Just leave the XML path as empty